_targets.R

```diff
@@ -0,0 +1,49 @@
+library(targets)
+
+options(tidyverse.quiet = TRUE)
+tar_option_set(packages = c("tidyverse", "lubridate","rmarkdown","dataRetrieval","knitr","leaflet","sf","sbtools")) 
+
+source("1_fetch.R")
+source("2_process.R")
+source("3_visualize.R")
+
+dir.create("1_fetch/out/", showWarnings = FALSE)
+dir.create("2_process/out/", showWarnings = FALSE)
+dir.create("3_visualize/out/", showWarnings = FALSE)
+
+# Define columns of interest from harmonized WQP data
+wqp_vars_select <- c("MonitoringLocationIdentifier","MonitoringLocationName","LongitudeMeasure","LatitudeMeasure",
+                     "MonitoringLocationTypeName","OrganizationIdentifier","ActivityStartDate","ActivityStartTime.Time",
+                     "ActivityEndDate","CharacteristicName","param_group","param","USGSPCode","ActivityMediaName",
+                     "ResultSampleFractionText","HydrologicCondition","HydrologicEvent","resultVal2","resultUnits2",
+                     "ResultDetectionConditionText","ResultTemperatureBasisText","PrecisionValue","ResultStatusIdentifier",
+                     "final")
+
+# Define WQP CharacteristicNames of interest
+# others: "Dissolved oxygen saturation, field, max","Dissolved oxygen saturation, field, min","Dissolved oxygen, field, max","Dissolved oxygen, field, min")
+CharNames_select = c("Dissolved oxygen (DO)","Dissolved oxygen saturation")
+params_select = c("Dissolved oxygen","Dissolved oxygen saturation","Dissolved oxygen saturation, field",
+                  "Dissolved oxygen, field","Dissolved oxygen, field, mean")
+
+# Define DO units of interest
+units_select = c("mg/l")
+
+# Define hydrologic event types in harmonized WQP data to exclude
+omit_wqp_events <- c("Spill","Volcanic action")
+
+# Define USGS parameter codes
+pcode_select <- c("00300") 
+
+# Define minor HUCs (hydrologic unit codes) that make up the DRB
+drb_huc8s <- c("02040101","02040102","02040104","02040103","02040106","02040105",
```

Comment:
Maybe here we can just do the subset of the lower DRB (#13)?

Reply:
Sure, we could subset the pipeline to only include the lower DRB. I added a comment to issue #13 earlier because I wasn't sure if the more extensive dataset would be helpful to retain for modeling.

Reply:
I think we could go either way. My opinion is that we should make the decision to bring in outside data later on and start with only data from our study area.

Reply:
OK, sounds good! I've gone ahead and subset the data to only include the lower DRB